src/App.js

```diff
@@ -8,8 +14,11 @@ function App() {
       {/* the Header component needs a string passed in as the `name` prop */}
       {/* the Footer component needs a year passed in as the `year` prop */}
       {/* pass the zodiac array imported above to the Main component as a `zodiacs` prop */}
+      <Header name='QPM'/>
+      <Main zodiac={ zodiac }/>
```

Comment:
Here you are correctly grabbing the zodiac array and passing it as a prop to the Main component.  But look down at the Main component....